### PR TITLE
DM-38942: Improve rho statistics documentation

### PR DESCRIPTION
**Pull request by arunkannawadi**
This adds an example plot and introduces appropriate subscripts in LaTeX format.

### CODE REVIEW

File: python/lsst/analysis/tools/actions/plot/rhoStatisticsPlot.py [hunk 1]
```diff
@@ -39,6 +39,13 @@
 
 
 class RhoStatisticsPlotAction(PlotAction):
+    """Make multiple plots of rho statistics.
+
+    Rho statistics capture the spatial correlation among various PSF size and
```
Comment (sr525) [inline python/lsst/analysis/tools/actions/plot/rhoStatisticsPlot.py:44]: amongst?

**Review by sr525 — APPROVED** [review]

File: python/lsst/analysis/tools/actions/vector/calcShapeSize.py [hunk 2]
```diff
@@ -65,9 +69,10 @@ class CalcShapeSize(VectorAction):
     sizeType = ChoiceField[str](
         doc="The type of size to calculate",
         default="determinant",
+        optional=False,
         allowed={
-            "trace": "trace radius",
-            "determinant": "determinant radius",
+            "trace": r"Trace radius :math:`\sqrt{(I_{xx}+I_{yy})/2}`",
+            "determinant": r"Determinant radius :math:`(I_{xx}I_{yy}-I_{xy}^2)^{\frac{1}{4}}`",
```
Comment (arunkannawadi) [inline python/lsst/analysis/tools/actions/vector/calcShapeSize.py:75]: image

File: python/lsst/analysis/tools/actions/vector/calcRhoStatistics.py [hunk 3]
```diff
@@ -238,52 +238,66 @@ class CalcRhoStatistics(KeyedDataAction):
...
+           et. al (2016).
            The DES Science Verification weak lensing shear catalogues
            MNRAS, 460, 2245–2281.
            https://doi.org/10.1093/mnras/stw990;
            https://arxiv.org/abs/1507.05603
     .. [2] Melchior, P., et. al (2015)
-           Mass and galaxy distributions of four massive galaxy clusters from Dark Energy Survey
-           Science Verification data
+           Mass and galaxy distributions of four massive galaxy clusters from
+           Dark Energy Survey Science Verification data
```
Comment (arunkannawadi) [inline python/lsst/analysis/tools/actions/vector/calcRhoStatistics.py:300]: image

File: python/lsst/analysis/tools/actions/vector/calcRhoStatistics.py [hunk 4]
```diff
@@ -314,8 +328,10 @@ class CalcRhoStatistics(KeyedDataAction):
     ellipticityType = ChoiceField[str](
         doc="The type of ellipticity to calculate",
         allowed={
-            "distortion": "Distortion, measured as (Ixx - Iyy)/(Ixx + Iyy)",
-            "shear": ("Shear, measured as (Ixx - Iyy)/(Ixx + Iyy + 2*sqrt(Ixx*Iyy - Ixy**2))"),
+            "distortion": r"Distortion, measured as :math:`(I_{xx}-I_{yy})/(I_{xx}+I_{yy})`",
+            "shear": (
+                r"Shear, measured as :math:`(I_{xx}-I_{yy})/(I_{xx}+I_{yy}+2\sqrt(I_{xx}I_{yy}-I_{xy}^2))`"
+            ),
```
Comment (arunkannawadi) [inline python/lsst/analysis/tools/actions/vector/calcRhoStatistics.py:334]: image

Comment (arunkannawadi) [reply python/lsst/analysis/tools/actions/vector/calcRhoStatistics.py:334]: Blah, missed some curly braces here.

File: python/lsst/analysis/tools/actions/vector/ellipticity.py [hunk 5]
```diff
@@ -171,12 +188,14 @@ class CalcEDiff(VectorAction):
...
-    .. math:: |de|*\exp{(j*theta)}.
-
-    If `halvePhaseAngle` config parameter is set to `True`, then
-    the returned quantity therefore corresponds to :math:`|e|*\exp{(j*theta)}`.
+    3. For plotting purposes we might want to plot quivers whose lengths
+    are proportional to :math:`|\delta e|` and whose angles correspond to
+    :math:`\theta_\delta`.
+    If `halvePhaseAngle` config parameter is set to `True`, then the returned
+    quantity therefore corresponds to the complex quantity
+    :math:`|\delta e|\exp{(\mathrm{i}\theta_\delta)}`.
```
Comment (arunkannawadi) [inline python/lsst/analysis/tools/actions/vector/ellipticity.py:198]: image

File: python/lsst/analysis/tools/actions/vector/ellipticity.py [hunk 6]
```diff
@@ -197,10 +216,11 @@ class CalcEDiff(VectorAction):
...
     component = ChoiceField[str](
         doc="Which component of the ellipticity to return. If `None`, return complex ellipticity values.",
-        optional=True,
         allowed={
-            "1": "e1 or g1 (depending on the `ellipiticyType`)",
-            "2": "e2 or g2 (depending on the `ellipiticyType`)",
+            "1": r":math:`\delta e_1` or :math:`\delta g_1` (depending on the common `ellipiticityType`)",
+            "2": r":math:`\delta e_2` or :math:`\delta g_2` (depending on the common `ellipiticityType`)",
+            None: r":math:`\delta e_1+\mathrm{i}\delta e_2` or :math:`\delta g_1 \mathrm{i}\delta g_2`"
+            " (depending on the common `ellipticityType`)",
```
Comment (arunkannawadi) [inline python/lsst/analysis/tools/actions/vector/ellipticity.py:223]: image

File: python/lsst/analysis/tools/actions/plot/rhoStatisticsPlot.py (hunk 1 above)
Comment (arunkannawadi) [reply python/lsst/analysis/tools/actions/plot/rhoStatisticsPlot.py:44]: image

File: python/lsst/analysis/tools/actions/plot/rhoStatisticsPlot.py [hunk 7]
```diff
@@ -114,6 +121,23 @@ def makePlot(
...
+            :math:`\rho_3'` is accessed with the key `rho3alt`.
+
+        Examples
+        --------
+        An example rho statistics plot may be seen below:
+
+        .. image:: /_static/analysis_tools/rhoPlotExample.png
+
+        For further details on how to generate a plot, please refer to the
+        :ref:`getting started guide<analysis-tools-getting-started>`.
```
Comment (arunkannawadi) [inline python/lsst/analysis/tools/actions/plot/rhoStatisticsPlot.py:140]: image

**Pull request merged**